src/vorta/models.py

```diff
@@ -215,6 +218,11 @@ def get_misc_settings():
             'label': trans_late('settings',
                                 'Open main window on startup')
         },
+        {
+            'key': 'get_srcpath_datasize', 'value': True, 'type': 'checkbox',
+            'label': trans_late('settings',
+                                'Get size of file/folder when added')
```

Comment:
```suggestion
                                'Get statistics of file/folder when added')
```
Since its size + count.